rsconf/package_data/vm_devbox/main.sh.jinja

```diff
@@ -4,16 +4,12 @@ vm_devbox_main() {
     if vagrant --version > /dev/null 2>&1; then
         return
     fi
-    declare p=kernel-devel-$(uname -r)
-    if ! yum list "$p" &> /dev/null; then
-       install_err "rpm $p not found.
-Virtualbox needs the kernel-devel rpm for the host kernel to be installed.
-Maybe try updating the kernel? The repos only have kernel-devel for recent versions of the kernel."
-    fi
-    rsconf_yum_install "$p"
-    yum-config-manager --add-repo https://download.virtualbox.org/virtualbox/rpm/el/virtualbox.repo
-    yum makecache -y
-    rsconf_yum_install VirtualBox-7.0
+    yum-config-manager --set-enabled crb
+    rsconf_yum_install '@Virtualization Hypervisor' '@Virtualization Tools' '@Development Tools' 'libvirt-devel'
     yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
     rsconf_yum_install vagrant
+    systemctl enable --now libvirtd
+    systemctl start libvirtd
+    # TODO(e-carlin):  discuss with rjn: security implications and rsconf way of doing this
+    usermod -aG libvirt vagrant
```

Comment:
@robnagler obviously this isn't right. Should at least use run_u var. But, how do you feel about this. Adding the run user is required to start the vm. Maybe there are ways around it. I'm not sure what all of this security implications are. Also, if we do keep it is there an "rsconf way" to add a user to a group?

Reply:
I don't think there's a security implication. It's better that the user be run_u than root.

Reply:
48af956